packages/common/src/service.js

```diff
@@ -52,9 +62,24 @@ function Service(config, options) {
     email: options.email
   });
 
+  // Some services have not been moved to the
```

Comment:
I think we should finish extending support for the new way before we leave ourselves in an incomplete state. Sorry that this didn't seem like a problem before, but since this concept has evolved to support new configurations, I don't think we should leave it incomplete. If you need a break or don't feel like completing it, no worries at all.

Reply:
I have no problem wrapping up the other services. I tend to dislike large PR so I don't want to put other maintainers in that position, however if you think it's necessary I can continue with the ones missing.

Reply:
I don't mind-- we've gotten pretty immune to massive PRs after being such a huge monorepo :)